index.js

```diff
@@ -171,7 +217,8 @@ async function main() {
     .option('-f, --force', 'Force push to remote')
     .option('--push-merged', 'Push all branches (inclusing those that have already been merged into master)')
     .option('--remote <remote>', 'Set remote to push to. Defaults to "origin"')
-    .option('-c, --create-prs', 'Create GitHub PRs from your train branches');
+    .option('-c, --create-prs', 'Create GitHub PRs from your train branches')
+    .option('-n, --new-branch <branch>', 'Create a new branch to the train and place it after the current branch');
```

Comment:
Should this rather be `-a/--add` because we're not creating a new branch but rather adding the current (existing) branch to the pr-train?

Reply:
Oh wait, I misunderstood. So this actually _does_ create a new branch and add it to the train, gotcha.